### ISSUE TITLE
[Bug]: NullReferenceException in `CKAN.RepositoryDataManager.<>c__DisplayClass7_0.<Update>b__1(Repository r)`

### ISSUE THREAD

**Issue by BeamingGato** [KSP-CKAN/CKAN#4107]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Operating System

Windows 10

### CKAN Version

1.34.4

### Games

KSP 1

### Game Version

1.12.5 latest

### Did you make any manual changes to your game folder (i.e., not via CKAN)?

No

### Describe the bug

![Screenshot 2024-06-14 175234](https://github.com/KSP-CKAN/CKAN/assets/172548112/a92d5238-946a-4d84-8aa6-d0bcb3977d04)

It won't let me load anything. I know this is probably going to be marked as a duplicate, by my case is a lot more severe than others'.

### Steps to reproduce

Open Ckan.exe
wait for it to update
wait for it to say, "repository update failed!"

### Relevant log output

```
Scanning for DLCs and manually installed modules...
Loading cached repository data
Updating repositories...
Refreshing game version data
Checking for updates
Object reference not set to an instance of an object.
Repository update failed!
```

**Comment by BeamingGato** [comment]
83 INFO  CKAN.CmdLine.MainClass  CKAN started.
   220 INFO  CKAN.CmdLine.MainClass  CKAN exiting.
   237 INFO  CKAN.CmdLine.MainClass  CKAN exiting.
   343 INFO  CKAN.GameInstanceManager  Loading KSP instances
   345 DEBUG CKAN.GameInstanceManager  Loading Kerbal Space Program from C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program
   359 DEBUG CKAN.Games.KerbalSpaceProgram.GameVersionProviders.KspBuildMap  Getting cached build map
   365 DEBUG CKAN.GameInstance  Found version 1.12.5.3190
   367 INFO  CKAN.GameInstance  Initialising C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN
   369 INFO  CKAN.GameInstance  Initialised C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN
   433 INFO  CKAN.GUI.Main  Starting the GUI
   878 DEBUG CKAN.GameInstance  Checking if KSP is in my current dir: C:\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program\CKAN
   878 DEBUG CKAN.GameInstance  Checking if KSP is in my exe dir: C:\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program\CKAN
   879 DEBUG CKAN.GameInstance  Checking if KSP2 is in my current dir: C:\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program\CKAN
   880 DEBUG CKAN.GameInstance  Checking if KSP2 is in my exe dir: C:\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program\CKAN
   906 DEBUG CKAN.GUI.Util  Found screen: {X=-1920,Y=0,Width=1920,Height=1080}
   906 DEBUG CKAN.GUI.Util  Clamped location: {X=-1635,Y=408}
   929 INFO  CKAN.GUI.URLHandlers  Adding URL handler to registry
   964 INFO  CKAN.GUI.Main  GUI started
  1038 DEBUG CKAN.RegistryManager  Preparing to load registry at C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN
  1040 DEBUG CKAN.RegistryManager  Checking for stale lock file at C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN\registry.locked
  1040 DEBUG CKAN.RegistryManager  Trying to create lock file: C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN\registry.locked
  1041 DEBUG CKAN.RegistryManager  Lock file created: C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN\registry.locked
  1042 DEBUG CKAN.RegistryManager  Trying to load registry from C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN\registry.json
  1043 DEBUG CKAN.RegistryManager  Registry JSON loaded; parsing...
  1306 DEBUG CKAN.RegistryManager  Registry loaded and parsed
  1306 INFO  CKAN.RegistryManager  Loaded CKAN registry at C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN\registry.json
  1363 INFO  CKAN.GUI.Main  Making auto-update call
  1364 DEBUG CKAN.Net  About to download https://api.github.com/repos/KSP-CKAN/CKAN/releases/latest
  1641 DEBUG CKAN.Net  Response from https://api.github.com/repos/KSP-CKAN/CKAN/releases/latest:

Vary: Accept, Accept-Encoding, Accept, X-Requested-With
X-GitHub-Media-Type: github.v3; format=json
x-github-api-version-selected: 2022-11-28
Access-Control-Expose-Headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 0
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
X-RateLimit-Limit: 60
X-RateLimit-Remaining: 59
X-RateLimit-Reset: 1718409540
X-RateLimit-Resource: core
X-RateLimit-Used: 1
Transfer-Encoding: chunked
X-GitHub-Request-Id: C534:593B2:858A35:E192EB:666CCB34
Accept-Ranges: bytes
Cache-Control: public, max-age=60, s-maxage=60
Content-Type: application/json; charset=utf-8
Date: Fri, 14 Jun 2024 22:59:00 GMT
ETag: W/"8703cf6d84e501d29656287a2e1afa78a6c1f65a052f1ea4c65425f244ac3d3a"
Last-Modified: Thu, 08 Feb 2024 00:09:36 GMT
Server: GitHub.com


{"url":"https://api.github.com/repos/KSP-CKAN/CKAN/releases/135871692","assets_url":"https://api.github.com/repos/KSP-CKAN/CKAN/releases/135871692/assets","upload_url":"https://uploads.github.com/repos/KSP-CKAN/CKAN/releases/135871692/assets{?name,label}","html_url":"https://github.com/KSP-CKAN/CKAN/releases/tag/v1.34.4","id":135871692,"author":{"login":"HebaruSan","id":1559108,"node_id":"MDQ6VXNlcjE1NTkxMDg=","avatar_url":"https://avatars.githubusercontent.com/u/1559108?v=4","gravatar_id":"","url":"https://api.github.com/users/HebaruSan","html_url":"https://github.com/HebaruSan","followers_url":"https://api.github.com/users/HebaruSan/followers","following_url":"https://api.github.com/users/HebaruSan/following{/other_user}","gists_url":"https://api.github.com/users/HebaruSan/gists{/gist_id}","starred_url":"https://api.github.com/users/HebaruSan/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/HebaruSan/subscriptions","organizations_url":"https://api.github.com/users/HebaruSan/orgs","repos_url":"https://api.github.com/users/HebaruSan/repos","events_url":"https://api.github.com/users/HebaruSan/events{/privacy}","received_events_url":"https://api.github.com/users/HebaruSan/received_events","type":"User","site_admin":false},"node_id":"RE_kwDOAV-5Ys4IGTzM","tag_name":"v1.34.4","target_commitish":"master","name":"Niven","draft":false,"prerelease":false,"created_at":"2024-01-04T18:27:52Z","published_at":"2024-01-04T18:35:53Z","assets":[{"url":"https://api.github.com/repos/KSP-CKAN/CKAN/releases/assets/143805739","id":143805739,"node_id":"RA_kwDOAV-5Ys4Ikk0r","name":"AutoUpdater.exe","label":"","uploader":{"login":"github-actions[bot]","id":41898282,"node_id":"MDM6Qm90NDE4OTgyODI=","avatar_url":"https://avatars.githubusercontent.com/in/15368?v=4","gravatar_id":"","url":"https://api.github.com/users/github-actions%5Bbot%5D","html_url":"https://github.com/apps/github-actions","followers_url":"https://api.github.com/users/github-actions%5Bbot%5D/followers","following_url":"https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}","gists_url":"https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}","starred_url":"https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/github-actions%5Bbot%5D/subscriptions","organizations_url":"https://api.github.com/users/github-actions%5Bbot%5D/orgs","repos_url":"https://api.github.com/users/github-actions%5Bbot%5D/repos","events_url":"https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}","received_events_url":"https://api.github.com/users/github-actions%5Bbot%5D/received_events","type":"Bot","site_admin":false},"content_type":"application/vnd.microsoft.portable-executable","state":"uploaded","size":423936,"download_count":40569,"created_at":"2024-01-04T18:38:39Z","updated_at":"2024-01-04T18:38:40Z","browser_download_url":"https://github.com/KSP-CKAN/CKAN/releases/download/v1.34.4/AutoUpdater.exe"},{"url":"https://api.github.com/repos/KSP-CKAN/CKAN/releases/assets/143805736","id":143805736,"node_id":"RA_kwDOAV-5Ys4Ikk0o","name":"ckan-1.34.4-1.noarch.rpm","label":"","uploader":{"login":"github-actions[bot]","id":41898282,"node_id":"MDM6Qm90NDE4OTgyODI=","avatar_url":"https://avatars.githubusercontent.com/in/15368?v=4","gravatar_id":"","url":"https://api.github.com/users/github-actions%5Bbot%5D","html_url":"https://github.com/apps/github-actions","followers_url":"https://api.github.com/users/github-actions%5Bbot%5D/followers","following_url":"https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}","gists_url":"https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}","starred_url":"https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/github-actions%5Bbot%5D/subscriptions","organizations_url":"https://api.github.com/users/github-actions%5Bbot%5D/orgs","repos_url":"https://api.github.com/users/github-actions%5Bbot%5D/repos","events_url":"https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}","received_events_url":"https://api.github.com/users/github-actions%5Bbot%5D/received_events","type":"Bot","site_admin":false},"content_type":"application/x-rpm","state":"uploaded","size":1529275,"download_count":1920,"created_at":"2024-01-04T18:38:38Z","updated_at":"2024-01-04T18:38:39Z","browser_download_url":"https://github.com/KSP-CKAN/CKAN/releases/download/v1.34.4/ckan-1.34.4-1.noarch.rpm"},{"url":"https://api.github.com/repos/KSP-CKAN/CKAN/releases/assets/143805731","id":143805731,"node_id":"RA_kwDOAV-5Ys4Ikk0j","name":"CKAN.dmg","label":"","uploader":{"login":"github-actions[bot]","id":41898282,"node_id":"MDM6Qm90NDE4OTgyODI=","avatar_url":"https://avatars.githubusercontent.com/in/15368?v=4","gravatar_id":"","url":"https://api.github.com/users/github-actions%5Bbot%5D","html_url":"https://github.com/apps/github-actions","followers_url":"https://api.github.com/users/github-actions%5Bbot%5D/followers","following_url":"https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}","gists_url":"https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}","starred_url":"https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/github-actions%5Bbot%5D/subscriptions","organizations_url":"https://api.github.com/users/github-actions%5Bbot%5D/orgs","repos_url":"https://api.github.com/users/github-actions%5Bbot%5D/repos","events_url":"https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}","received_events_url":"https://api.github.com/users/github-actions%5Bbot%5D/received_events","type":"Bot","site_admin":false},"content_type":"application/x-apple-diskimage","state":"uploaded","size":5013504,"download_count":4411,"created_at":"2024-01-04T18:38:37Z","updated_at":"2024-01-04T18:38:37Z","browser_download_url":"https://github.com/KSP-CKAN/CKAN/releases/download/v1.34.4/CKAN.dmg"},{"url":"https://api.github.com/repos/KSP-CKAN/CKAN/releases/assets/143805726","id":143805726,"node_id":"RA_kwDOAV-5Ys4Ikk0e","name":"ckan.exe","label":"","uploader":{"login":"github-actions[bot]","id":41898282,"node_id":"MDM6Qm90NDE4OTgyODI=","avatar_url":"https://avatars.githubusercontent.com/in/15368?v=4","gravatar_id":"","url":"https://api.github.com/users/github-actions%5Bbot%5D","html_url":"https://github.com/apps/github-actions","followers_url":"https://api.github.com/users/github-actions%5Bbot%5D/followers","following_url":"https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}","gists_url":"https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}","starred_url":"https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/github-actions%5Bbot%5D/subscriptions","organizations_url":"https://api.github.com/users/github-actions%5Bbot%5D/orgs","repos_url":"https://api.github.com/users/github-actions%5Bbot%5D/repos","events_url":"https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}","received_events_url":"https://api.github.com/users/github-actions%5Bbot%5D/received_events","type":"Bot","site_admin":false},"content_type":"application/vnd.microsoft.portable-executable","state":"uploaded","size":4875264,"download_count":119489,"created_at":"2024-01-04T18:38:36Z","updated_at":"2024-01-04T18:38:36Z","browser_download_url":"https://github.com/KSP-CKAN/CKAN/releases/download/v1.34.4/ckan.exe"},{"url":"https://api.github.com/repos/KSP-CKAN/CKAN/releases/assets/143805734","id":143805734,"node_id":"RA_kwDOAV-5Ys4Ikk0m","name":"ckan_1.34.4_all.deb","label":"","uploader":{"login":"github-actions[bot]","id":41898282,"node_id":"MDM6Qm90NDE4OTgyODI=","avatar_url":"https://avatars.githubusercontent.com/in/15368?v=4","gravatar_id":"","url":"https://api.github.com/users/github-actions%5Bbot%5D","html_url":"https://github.com/apps/github-actions","followers_url":"https://api.github.com/users/github-actions%5Bbot%5D/followers","following_url":"https://api.github.com/users/github-actions%5Bbot%5D/following{/other_user}","gists_url":"https://api.github.com/users/github-actions%5Bbot%5D/gists{/gist_id}","starred_url":"https://api.github.com/users/github-actions%5Bbot%5D/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/github-actions%5Bbot%5D/subscriptions","organizations_url":"https://api.github.com/users/github-actions%5Bbot%5D/orgs","repos_url":"https://api.github.com/users/github-actions%5Bbot%5D/repos","events_url":"https://api.github.com/users/github-actions%5Bbot%5D/events{/privacy}","received_events_url":"https://api.github.com/users/github-actions%5Bbot%5D/received_events","type":"Bot","site_admin":false},"content_type":"application/vnd.debian.binary-package","state":"uploaded","size":1052296,"download_count":2101,"created_at":"2024-01-04T18:38:38Z","updated_at":"2024-01-04T18:38:38Z","browser_download_url":"https://github.com/KSP-CKAN/CKAN/releases/download/v1.34.4/ckan_1.34.4_all.deb"}],"tarball_url":"https://api.github.com/repos/KSP-CKAN/CKAN/tarball/v1.34.4","zipball_url":"https://api.github.com/repos/KSP-CKAN/CKAN/zipball/v1.34.4","body":"[![Niven ring](https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/Ringworld_drawing.jpg/640px-Ringworld_drawing.jpg)](https://en.wikipedia.org/wiki/Known_Space)\r\n\r\n---\r\n\r\n## Changes since v1.34.2\r\n\r\n### Features\r\n\r\n- [CLI] Pause after administrator error message (#3966 by @HebaruSan; reviewed: techman83)\r\n- [Multiple] Put auto-installed mods in ignored modpack group by default (#3978 by @HebaruSan)\r\n- [GUI] Add links to hidden tags and labels below mod list (#3979, #3980 by @HebaruSan)\r\n- [Core] Detect and use Windows KSP2 executable on Linux (#3984 by @JackOfHertz; reviewed: HebaruSan)\r\n\r\n### Bugfixes\r\n\r\n- [Core] Oops, `HttpClient` actually sucks (#3960 by @HebaruSan; reviewed: techman83)\r\n- [Multiple] Usability improvements for adding game instance (#3964 by @HebaruSan; reviewed: JonnyOThan)\r\n- [Core] Fix NullReferenceException in csv/tsv export (#3967 by @HebaruSan)\r\n- [Core] Fix cache timestamp comparisons (#3974 by @HebaruSan)\r\n- [GUI] Fix compatible popup messing with max game version column (#3976 by @HebaruSan)\r\n- [Core] Fix Audit Recommendations option (#3988 by @HebaruSan)\r\n- [Multiple] Installation message usability improvements (#3989 by @HebaruSan)\r\n\r\n### Internal\r\n\r\n- [Netkan] Re-sort module properties after merging (#3957 by @HebaruSan)\r\n- [Netkan] Update and optimize meta tester and inflator images (#3958, #3968 by @HebaruSan; reviewed: techman83)\r\n- [Netkan] Generate schema compliant game versions from swinfo.json (#3969 by @HebaruSan)\r\n- [Core] Save and load game version for fake KSP2 instances (#3986 by @HebaruSan)\r\n- [Netkan] Resolve compatibility conflicts after multi-host merge (#3991 by @HebaruSan)\r\n\r\n## Notes\r\n\r\n- CKAN releases are built in clean-room conditions to protect against viruses. If your virus scanner reports a problem, it's almost certainly a false positive. Please report such problems to the company that produces your virus scanner so they can update their whitelisting.\r\n- This release of the CKAN has not been tested on Mono releases prior to 6.8. We highly recommend that Mac and Linux users upgrade to the latest stable release of Mono from [mono-project.com](http://www.mono-project.com/). You will need the equivalent of the mono-complete package for your OS.\r\n\r\n## How should I install?\r\n\r\nOS | Description\r\n:-- | :--\r\nWindows | Download **ckan.exe** and run it. Make sure you have [.NET 4.8](https://dotnet.microsoft.com/en-us/download/dotnet-framework/net48) or later installed.\r\nmacOS | Get the **.dmg**, open it, copy **CKAN.app** somewhere and run it\r\nDebian, Ubuntu, Linux Mint | [Install from our APT repo for automated updates](https://github.com/KSP-CKAN/CKAN/blob/master/doc/apt-repo.md), or get the **.deb** file and install it with dpkg-install/apt-get/apt\r\nFedora, openSUSE | [Install from our RPM repo for automated updates](https://github.com/KSP-CKAN/CKAN/blob/master/doc/rpm-repo.md), or get the **.rpm** file and install it with rpm/yum/dnf/zypper\r\nArch, Manjaro | CKAN is in the [Arch User Repository](https://aur.archlinux.org/packages/ckan/). If Arch's MSBuild breaks again, get ckan.exe and run it with Mono.\r\nOther | Get **ckan.exe** and run it with Mono\r\n\r\n**Nobody** needs to download AutoUpdate.exe.\r\n\r\n---\r\n\r\nRelease image under CC-BY-3.0, [courtesy of Eric M. Jones](https://commons.wikimedia.org/wiki/File:Ringworld_drawing.jpg)","reactions":{"url":"https://api.github.com/repos/KSP-CKAN/CKAN/releases/135871692/reactions","total_count":8,"+1":7,"-1":0,"laugh":1,"hooray":0,"confused":0,"heart":0,"rocket":0,"eyes":0},"mentions_count":2}
  1694 DEBUG CKAN.GUI.Main  Current instance updated, scanning
  1750 DEBUG CKAN.GUI.Main  Scanning before repo update
  1751 INFO  CKAN.RegistryManager  Scanning for installed mods...
  1752 DEBUG CKAN.CkanTransaction  Trying to set max timeout to 49:17:02:47.294
  1755 DEBUG CKAN.CkanTransaction  Failed to set s_cachedMaxTimeout
  1755 DEBUG CKAN.CkanTransaction  Failed to set s_maximumTimeout
  1756 DEBUG CKAN.CkanTransaction  Starting transaction with timeout 49:17:02:47.294
  1870 DEBUG CKAN.RegistryManager  Registering DLLs: 
  1873 DEBUG CKAN.Registry  Enlisting registry with tx 8c7f4331-534d-4841-9b38-c2a39fbba86f:1
  1885 DEBUG CKAN.Registry  State saved
  1886 DEBUG CKAN.Registry  Invalidating installed mod caches
  1897 DEBUG CKAN.Registry  Invalidating installed mod caches
  1898 DEBUG CKAN.Registry  Trying to find BreakingGround-DLC version 1.7.1 (unmanaged)
  1902 DEBUG CKAN.RepositoryDataManager  Looking for data in C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  1904 DEBUG CKAN.RepositoryData  Trying to load repository data from C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  2909 DEBUG CKAN.RepositoryData  Valid repository data not found at C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json: Unterminated string. Expected delimiter: ". Path 'available_modules.MK2KSPIIntegration.module_version['8.6.0_Perfectly_Ultimate_Great_Main_Sequence_Panthera_leo'].resources.homepage', line 915875, position 8375185.
  2912 DEBUG CKAN.Registry  Trying to find MakingHistory-DLC version 1.12.1 (unmanaged)
  2913 DEBUG CKAN.RepositoryDataManager  Looking for data in C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  2913 DEBUG CKAN.RepositoryData  Trying to load repository data from C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  4230 DEBUG CKAN.RepositoryData  Valid repository data not found at C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json: Unterminated string. Expected delimiter: ". Path 'available_modules.MK2KSPIIntegration.module_version['8.6.0_Perfectly_Ultimate_Great_Main_Sequence_Panthera_leo'].resources.homepage', line 915875, position 8375185.
  4231 DEBUG CKAN.RegistryManager  Scan completed, committing transaction
  4231 DEBUG CKAN.Registry  Registry prepared to commit transaction
  4232 DEBUG CKAN.Registry  Committing registry tx 8c7f4331-534d-4841-9b38-c2a39fbba86f:1
  4235 DEBUG CKAN.RepositoryDataManager  Looking for data in C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  4235 DEBUG CKAN.RepositoryData  Trying to load repository data from C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  5417 DEBUG CKAN.RepositoryData  Valid repository data not found at C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json: Unterminated string. Expected delimiter: ". Path 'available_modules.MK2KSPIIntegration.module_version['8.6.0_Perfectly_Ultimate_Great_Main_Sequence_Panthera_leo'].resources.homepage', line 915875, position 8375185.
  5421 DEBUG CKAN.RepositoryDataManager  Looking for data in C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  5421 DEBUG CKAN.RepositoryData  Trying to load repository data from C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  6764 DEBUG CKAN.RepositoryData  Valid repository data not found at C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json: Unterminated string. Expected delimiter: ". Path 'available_modules.MK2KSPIIntegration.module_version['8.6.0_Perfectly_Ultimate_Great_Main_Sequence_Panthera_leo'].resources.homepage', line 915875, position 8375185.
  6782 DEBUG CKAN.RepositoryDataManager  Looking for data in C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  6782 DEBUG CKAN.RepositoryData  Trying to load repository data from C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json
  7995 DEBUG CKAN.RepositoryData  Valid repository data not found at C:\Users\Owner\AppData\Local\CKAN\repos\4A85405A-default.json: Unterminated string. Expected delimiter: ". Path 'available_modules.MK2KSPIIntegration.module_version['8.6.0_Perfectly_Ultimate_Great_Main_Sequence_Panthera_leo'].resources.homepage', line 915875, position 8375185.
  8021 DEBUG CKAN.CkanTransaction  Starting transaction with timeout 49:17:02:47.294
  8029 DEBUG CKAN.Games.KerbalSpaceProgram.GameVersionProviders.KspBuildMap  Refreshing build map from server
  8030 DEBUG CKAN.Games.KerbalSpaceProgram.GameVersionProviders.KspBuildMap  Getting remote build map
  8030 DEBUG CKAN.Net  About to download https://raw.githubusercontent.com/KSP-CKAN/CKAN-meta/master/builds.json
  8124 DEBUG CKAN.Net  Response from https://raw.githubusercontent.com/KSP-CKAN/CKAN-meta/master/builds.json:

Connection: keep-alive
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 4163:346FB:2C95A1:320EEB:666C40F8
X-Served-By: cache-bos4622-BOS
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1718405947.756585,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 5df9b33d871a5e2c9cc4597a45f9ab56602da45d
Source-Age: 57
Accept-Ranges: bytes
Content-Length: 2721
Cache-Control: max-age=300
Content-Type: text/plain; charset=utf-8
Date: Fri, 14 Jun 2024 22:59:06 GMT
Expires: Fri, 14 Jun 2024 23:04:06 GMT
ETag: "89fb689bd564c404807c41f50425a3326d3e969c54d34f6f1a2bdc8e2f5b5506"
Via: 1.1 varnish


{
  "builds": {
    "395": "0.23.0.395",
    "464": "0.23.5.464",
    "559": "0.24.2.559",
    "642": "0.25.0.642",
    "705": "0.90.0.705",
    "830": "1.0.0.830",
    "840": "1.0.1.840",
    "842": "1.0.2.842",
    "859": "1.0.3.859",
    "861": "1.0.4.861",
    "1024": "1.0.5.1024",
    "1028": "1.0.5.1028",
    "1172": "1.1.0.1172",
    "1174": "1.1.0.1174",
    "1180": "1.1.0.1180",
    "1183": "1.1.0.1183",
    "1196": "1.1.0.1196",
    "1203": "1.1.0.1203",
    "1209": "1.1.0.1209",
    "1215": "1.1.0.1215",
    "1224": "1.1.0.1224",
    "1228": "1.1.0.1228",
    "1230": "1.1.0.1230",
    "1250": "1.1.1.1250",
    "1260": "1.1.2.1260",
    "1289": "1.1.3.1289",
    "1473": "1.2.0.1473",
    "1479": "1.2.0.1479",
    "1485": "1.2.0.1485",
    "1486": "1.2.0.1486",
    "1489": "1.2.0.1489",
    "1494": "1.2.0.1494",
    "1499": "1.2.0.1499",
    "1500": "1.2.0.1500",
    "1509": "1.2.0.1509",
    "1517": "1.2.0.1517",
    "1520": "1.2.0.1520",
    "1523": "1.2.0.1523",
    "1532": "1.2.0.1532",
    "1539": "1.2.0.1539",
    "1540": "1.2.0.1540",
    "1546": "1.2.0.1546",
    "1548": "1.2.0.1548",
    "1553": "1.2.0.1553",
    "1563": "1.2.0.1563",
    "1564": "1.2.0.1564",
    "1569": "1.2.0.1569",
    "1574": "1.2.0.1574",
    "1576": "1.2.0.1576",
    "1583": "1.2.0.1583",
    "1584": "1.2.0.1584",
    "1586": "1.2.0.1586",
    "1604": "1.2.1.1604",
    "1622": "1.2.2.1622",
    "1727": "1.2.9.1727",
    "1730": "1.2.9.1730",
    "1737": "1.2.9.1737",
    "1738": "1.2.9.1738",
    "1743": "1.2.9.1743",
    "1750": "1.2.9.1750",
    "1758": "1.2.9.1758",
    "1764": "1.2.9.1764",
    "1773": "1.2.9.1773",
    "1781": "1.2.9.1781",
    "1790": "1.2.9.1790",
    "1796": "1.2.9.1796",
    "1800": "1.2.9.1800",
    "1804": "1.3.0.1804",
    "1836": "1.3.1.1836",
    "1847": "1.3.1.1847",
    "1855": "1.3.1.1855",
    "1863": "1.3.1.1863",
    "1891": "1.3.1.1891",
    "2077": "1.4.0.2077",
    "2089": "1.4.1.2089",
    "2110": "1.4.2.2110",
    "2152": "1.4.3.2152",
    "2215": "1.4.4.2215",
    "2243": "1.4.5.2243",
    "2256": "1.4.5.2256",
    "2332": "1.5.0.2332",
    "2335": "1.5.1.2335",
    "2395": "1.6.0.2395",
    "2401": "1.6.1.2401",
    "2483": "1.7.0.2483",
    "2539": "1.7.1.2539",
    "2555": "1.7.2.2555",
    "2556": "1.7.2.2556",
    "2594": "1.7.3.2594",
    "2686": "1.8.0.2686",
    "2694": "1.8.1.2694",
    "2781": "1.9.0.2781",
    "2788": "1.9.1.2788",
    "2917": "1.10.0.2917",
    "2939": "1.10.1.2939",
    "3045": "1.11.0.3045",
    "3066": "1.11.1.3066",
    "3077": "1.11.2.3077",
    "3140": "1.12.0.3140",
    "3142": "1.12.1.3142",
    "3167": "1.12.2.3167",
    "3173": "1.12.3.3173",
    "3187": "1.12.4.3187",
    "3190": "1.12.5.3190"
  }
}

  8131 ERROR CKAN.GUI.Main  Object reference not set to an instance of an object.
System.NullReferenceException: Object reference not set to an instance of an object.
   at CKAN.RepositoryDataManager.<>c__DisplayClass7_0.<Update>b__1(Repository r)
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at CKAN.RepositoryDataManager.Update(Repository[] repos, IGame game, Boolean skipETags, NetAsyncDownloader downloader, IUser user)
   at CKAN.GUI.Main.UpdateRepo(Object sender, DoWorkEventArgs e)
   at System.ComponentModel.BackgroundWorker.OnDoWork(DoWorkEventArgs e)
   at System.ComponentModel.BackgroundWorker.WorkerThreadStart(Object argument)
 11761 DEBUG CKAN.RegistryManager  Disposing of lock file writer at C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN\registry.locked
 11761 DEBUG CKAN.RegistryManager  Disposing of lock file stream at C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN\registry.locked
 11762 DEBUG CKAN.RegistryManager  Dispose of registry at C:/Program Files (x86)/Steam/steamapps/common/Kerbal Space Program/CKAN
 11796 INFO  CKAN.CmdLine.MainClass  CKAN exiting.

**Comment by HebaruSan** [comment]
Hi @BeamingGato, this looks the same as the issues fixed in #4077 and #4078. Can you please try the dev build and let us know if you still have this issue?

- <https://ksp-ckan.s3-us-west-2.amazonaws.com/ckan.exe>

Thanks!
